### PR TITLE
Improving doc extraction command

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Improving doc extraction command

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by mgdelacroix — APPROVED** [review]

**Review by reflog — CHANGES_REQUESTED** [review]

File: services/docextractor/plain.go [hunk 1]
```diff
@@ -36,11 +40,11 @@ func (pe *plainExtractor) Extract(filename string, r io.ReadSeeker) (string, err
 			break
 		}
 		count += size
-		if count > 1024 {
+		if count > total-4 || count > len(runes)-4 {
```
[the remark this answers is not in the document]
Comment (jespino) [reply services/docextractor/plain.go:43]: yes, you are right, a constant there can explain the intention behind that.

**Review by reflog — APPROVED** [review]

**Pull request merged**